engine/src/main/java/org/terasology/rendering/nui/TextLineBuilder.java

```diff
@@ -54,6 +54,10 @@ public TextLineBuilder(Font font, int maxWidth, int startOffset) {
     }
 
     public void addText(String text) {
+        if (text == null) {
```

Comment:
Suggest replace with precondition. Null should not be added - it is a bug that should be handled outside of this.